### ISSUE TITLE
500 error when trying to view a table with JSON formatted fields

### ISSUE THREAD

**Issue by abrenoch** [phpmyadmin/phpmyadmin#13273]
### Steps to reproduce
1. Create a new mysql table with a JSON formatted column
2. Click on 'browse'

### Expected behaviour
It should display the content (or lack thereof) of the newly created table

### Actual behaviour
Return a 500 internal server error

### Server configuration
**Operating system**: Windows server 2008

**Web server:** Apache/2.4.9 (Win32)

**Database:** Mysql 5.7.17

**PHP version:** php5.5.35 (nts) launched via fcgi through apache

**phpMyAdmin version:** 4.7.0

### Client configuration
**Browser:** Chrome 58

**Operating system:** Windows 7

-------

No errors are sent to the php log, but apache produces:
```
[Fri May 12 15:12:42.655592 2017] [fcgid:warn] [pid 2384:tid 1000] (OS 109)The pipe has been ended.  : [client 192.168.3.129:54368] mod_fcgid: get overlap result error
[Fri May 12 15:12:42.655592 2017] [core:error] [pid 2384:tid 1000] [client 192.168.3.129:54368] End of script output before headers: sql.php
```
Because of this I am inclined to say this is a FCGI problem, but I can't nail it down for the life of me!

Any help would be REALLY appreciated!

**Comment by nijel** [comment]
Are you using JSON type in MySQL, if yes, than you will have to upgrade PHP, see https://bugs.php.net/bug.php?id=70384 and https://github.com/phpmyadmin/phpmyadmin/issues/12364.

**Comment by abrenoch** [comment]
Yes I am using the JSON field in MySQL.. It looks like I would have to bumpPHP up to at least 5.6 to get this resolved (I'm on 5.5.35).

Thanks for the help!
